dali/python/nvidia/dali/pipeline.py

```diff
@@ -190,6 +190,8 @@ def __init__(self,
                  default_cuda_stream_priority=0,
                  *,
                  enable_memory_stats=False,
+                 checkpointing=False,
+                 checkpoint_to_restore=None,
```

Comment:
Please add both args to the docs and stress that this is experimental feature.

Reply:
That's a matter of preference I guess, but I'd go with enable_checkpointing and checkpoint names.